german/GUI_Automation/src/main/java/at05ui/core/selenium/WebDriverConfig.java

```diff
@@ -0,0 +1,63 @@
+package at05ui.core.selenium;
+
+import at05ui.core.utils.JsonReader;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+
+public class WebDriverConfig {
+    private Logger log = Logger.getLogger(getClass());
+
+    private static final String BROWSER = "browser";
+    private static final String BROWSER_NAME = System.getProperty("browserName");
```

Comment:
Where are you uisng this

Reply:
Here:
browser = (BROWSER.isEmpty()) ?
      configReader.getKeyValue(DRIVER, BROWSER) :
      BROWSER_NAME;

Reply:
It is correct, it had no use, but I use it later